config/community-resources.json

```
@@ -98,5 +98,15 @@
       "name": "Maximising Your Job Prospects in Tech with BioDrop",
       "url": "https://loftwah.medium.com/maximising-your-job-prospects-in-tech-with-BioDrop-2fea408301cc"
     }
+  },
+  {
+    "name": "Kim McMahon",
+    "username": "KimMchon",
```

This username doesn't exist, the username should be

```suggestion
    "username": "KimMcMahon",
```